metadata/ComputerVision__5__metadata.json

```diff
@@ -1,5 +1,5 @@
 {
-  "changeLog": "Release v2023.04",
+  "changeLog": "Release v2023.04.1",
```

Comment:
I think it would be cleaner to match the `changeLog` version to the `customVersion` - both to be with `23.04.2`